mne/annotations.py

```diff
@@ -0,0 +1,49 @@
+# Authors: Jaakko Leppakangas <jaeilepp@student.jyu.fi>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from datetime import datetime
+import time
+
+
+class Annotations(object):
+    """Annotation object for annotating segments of raw data.
+
+    Parameters
+    ----------
+    onset: array of float
```

Comment:
onset: array of float, shape (n_annotations,)

and put the shape below too to show that the size of all parameters must be the same